fpga/lib/ips/dma.cpp

```diff
@@ -9,6 +9,9 @@
 #include <sstream>
 #include <string>
 
+#include "xilinx/xaxidma_bd.h"
```

Comment:
Please use `<>` to actually indicate that the headers should be looked up in the compilers search path rather than a relative/absolute path.

Reply:
Also sorting the includes by lib/type would be nice :)